cdk/l3-cdk-resources/encryption-at-rest-express/README.md

```diff
@@ -0,0 +1,246 @@
+
+### Minimal configuration to use this app
+In this example, @mongodbatlas-awscdk/encryption-at-rest-express is used to enable encryption at rest for your MongoDB Atlas Project.
+
+```typescript
+import * as cdk from 'aws-cdk-lib';
+import { AtlasEncryptionAtRestExpress } from './index';
+
+const app = new cdk.App();
+const stack = new cdk.Stack(app, 'atlas-encryption-at-rest-express', {
+  env: { region: process.env.CDK_DEFAULT_REGION, account: process.env.CDK_DEFAULT_ACCOUNT },
+});
+
+
+const PROJECT_ID = stack.node.tryGetContext('MONGODB_ATLAS_PROJECT_ID') || process.env.MONGODB_ATLAS_PROJECT_ID;
+const ROLE_ID = stack.node.tryGetContext('MONGODB_ATLAS_ROLE_ID') || process.env.MONGODB_ATLAS_ROLE_ID;
+const MASTER_KEY_ID = stack.node.tryGetContext('MONGODB_ATLAS_MASTER_KEY_ID') || process.env.MONGODB_ATLAS_MASTER_KEY_ID;
+
+new AtlasEncryptionAtRestExpress(stack, 'atlas-encryption-at-rest-express', {
+  projectId: PROJECT_ID,
+
+  encryptionAtRest: {
+    roleId: ROLE_ID,
+    customerMasterKeyId: MASTER_KEY_ID,
+  },
+});
+```
+
+### Minimal configuration to create a cluster, database User, configure atlas access list and enable encryption at rest
+
+In this example we are providing the minimal configuration to the constructor to define an Atlas Cluster, DB user, add an entry to the Atlas Access list, and enable encryption at rest for the cluster and project.
+
+```typescript
+import * as cdk from 'aws-cdk-lib';
+import { AtlasEncryptionAtRestExpress } from './index';
+
+const app = new cdk.App();
+const stack = new cdk.Stack(app, 'atlas-encryption-at-rest-express', {
+  env: { region: process.env.CDK_DEFAULT_REGION, account: process.env.CDK_DEFAULT_ACCOUNT },
+});
+
+
+const PROJECT_ID = stack.node.tryGetContext('MONGODB_ATLAS_PROJECT_ID') || process.env.MONGODB_ATLAS_PROJECT_ID;
+const ROLE_ID = stack.node.tryGetContext('MONGODB_ATLAS_ROLE_ID') || process.env.MONGODB_ATLAS_ROLE_ID;
+const MASTER_KEY_ID = stack.node.tryGetContext('MONGODB_ATLAS_MASTER_KEY_ID') || process.env.MONGODB_ATLAS_MASTER_KEY_ID;
+const DB_USER_PASSWORD = stack.node.tryGetContext('MONGODB_ATLAS_DB_USER_PASSWORD') || process.env.MONGODB_ATLAS_DB_USER_PASSWORD;
+
+new AtlasEncryptionAtRestExpress(this, 'EncryptionAtRestExpress', {
+      cluster: {
+        name: "ClusterName",
+      },
+
+      accessList: {
+        accessList: [{
+          ipAddress: "192.0.0.1"
+        }]
+      },
+
+      encryptionAtRest: {
+        customerMasterKeyId: MONGODB_ATLAS_MASTER_KEY_ID,
+        roleId: ROLE_ID,
+      },
+
+      databaseUser: {
+        password: DB_USER_PASSWORD
+      },
+
+      projectId: PROJECT_ID,
+    })
+  }
+```
+The constructor will deploy the following resources using default values:
```

Comment:
customers have to provide the projectId, this constructor won't create the project. We cannot create the project since the user has to Enable Role-Based Access for a Project and we do not support the cloudProviderAccess endpoint via CFN.